modules/hcp-get-ocp-channel.adoc

```diff
@@ -2,134 +2,82 @@
 //
 // * hosted_control_planes/hcp-updating.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: PROCEDURE
 [id="hcp-get-ocp-channel_{context}"]
-= Setting channels in a hosted cluster
+= Updating your hosted cluster
 
-You can see available updates in the `HostedCluster.Status` field of the `HostedCluster` custom resource (CR).
+You can update the hosted cluster by setting a channel in your `HostedCluster` resource or by using the `oc patch` command.
 
-The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The list of the available updates can be different from the available updates from the following fields of the `HostedCluster` custom resource (CR):
+The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster. The initial `HostedCluster` resource does not have any information in the `status.version.availableUpdates` and `status.version.conditionalUpdates` fields. After you set the `spec.channel` field to the stable {product-title} release version, the HyperShift Operator reconciles the `HostedCluster` resource and updates the `status.version` field with the available and conditional updates.
```

Comment:
> The available updates are not fetched from the Cluster Version Operator (CVO) of a hosted cluster.

This sentence predates your pull request, but I'm not sure what it's trying to say.  When you set an OpenShift Update Service channel per these docs, that will propagate down to the CVO, which will calculate available updates, and those will propagate back up to HostedCluster's `status`.  Maybe we should drop or edit the sentence?